default_handler.go

```diff
@@ -38,12 +38,16 @@ type defaultHandler struct {
 	defaultIntf map[string]*exportedIntf
 }
 
-func (h *defaultHandler) PathExists(path ObjectPath) bool {
-	_, ok := h.objects[path]
-	return ok
+func (h *defaultHandler) PathExists(path ObjectPath) (*exportedObj, bool) {
```

Comment:
Would rename this method then since it not only checks for existence anymore

Reply:
Yes, the name should be changed, GetExportedObject? 
The sync.Map type, https://pkg.go.dev/sync#Map, does something similar and they call it Load(key) but I'm not sure it is better.

Reply:
I changed it to GetExportedObject since that is what it is.